python/tk_desktop/no_apps_installed_overlay.py

```diff
@@ -266,3 +269,15 @@ def _get_sg_software_entities(self, engine, project=None):
             return
 
         return sg_softwares
+
+    def __get_sg_server_version(self, engine):
+        """
+        Retrieves the shotgun server version
+        from the currently connected Shotgun.
+
+        :returns: Tuple of (major, minor, patch) versions.
+        """
+        sg_major_ver = engine.shotgun.server_info["version"][0]
+        sg_minor_ver = engine.shotgun.server_info["version"][1]
+        sg_patch_ver = engine.shotgun.server_info["version"][2]
+        return sg_major_ver, sg_minor_ver, sg_patch_ver
```

Comment:
Is it possible to just rely on ?

```suggestion
        return engine.shotgun.server_info["version"][:3]
```

Reply:
Picking individual index raise an IndexError if does not exist. While picking an index range returns whatever length is available. So it's not exactly the same behavior.

Besides, I copied/pasted this function from tk-multi-launchapp and I already created a ticket for refactoring this duplication into a framework.